photon-lib/src/main/java/org/photonvision/simulation/VideoSimUtil.java

```diff
@@ -374,8 +374,8 @@ public static void drawTagDetection(int id, Point[] dstPoints, Mat destination)
     /**
      * Set the field dimensions that are used for drawing the field wireframe.
      *
-     * @param fieldLengthMeters
-     * @param fieldWidthMeters
+     * @param fieldLengthMeters field length in meters
```

Comment:
This comment is not very helpful. Can we maybe also say how width/length correspond to X and Y?